### DISCUSSION TITLE
Student Access to Copilot

### DISCUSSION THREAD

**Discussion by karhu1** [community/community#52223] [GitHub Education] [answered]
My student developer pack request was reviewed and accepted, but when I go to access Copilot, the [website](https://github.com/features/copilot) only shows the default page with no student offer.

I know that Copilot is offered for free through the Student developer pack, so how can I gain access to it?

Here is an image of what my settings tab shows:

![image](https://user-images.githubusercontent.com/107146650/230534291-a412a16e-12ae-40cd-ba85-1a65499a3253.png)

**Comment by enjin1314** [comment]
<img width="480" alt="image" src="https://user-images.githubusercontent.com/64971822/230718923-3ae5f8cd-5441-4bcc-91d8-115899f135c3.png">
I think it's because you've logged into GitHub in vscode, please log out and try again.

**Comment by karhu1** [comment, marked as answer]
### What worked:

1. In your account settings, under the Copilot tab, click the "Start free trial" button.
2. That's it. Somehow that acts as the student dev pack button, and should give you the option for free Copilot without registering. 

If you already used the free trial, and you get a button to buy or subscribe to copilot, you should click that instead. it will then activate your student license, but only a week after the application for the student benefits was approved.

**Reply by LukeHebert** [reply]
Thanks, I had the same issue and this worked for me! Very unintuitive

**Reply by KoenBoone** [reply]
not a complete answer...
if you have already used the free trial before, you cannot click 'start free trial' anymore, or you can and it does nothing but refresh the page while nothing else happens.
If you already used the free trial, and you get a button to buy or subscribe to copilot, you should click that instead.  it will then activate your student license, atleast that worked for me... but only more than a week after my application for the student benefits was approved...

**Comment by KoenBoone** [comment]
not if you already used up the trial version

**Reply by haoliu001** [reply]
how should we solve it. I already used up the trail version, and then get github education, and now it does not work

**Reply by KoenBoone** [reply]
you can use the pay button, when you are accepted for the github education, it will just activate the free version, it's a bit of misleading, but just go ahead, don't have to give payment details at all

**Reply by haoliu001** [reply]
Sorry, I do not get the idea about the pay button, Where could I find it? (When I click the button "start free trial", it just refreshes and nothing changes.)

Is it in the billing and plans—plans and usage? 
<img width="1024" alt="image" src="https://github.com/community/community/assets/107664290/18783ca7-427b-4799-b412-67c9531c3f44">

**Reply by KoenBoone** [reply]
it is under your account (click on your avatar/photo in the top right corner) => settings => code,planning and automation => Copilot
there you can activate it, or buy it... sincy you are eligible, even if you only see a buy or pay button, click it and you will not be asked for payment details

**Reply by KoenBoone** [reply]
and then you should finaly see this after activating it... but in this screenshot you see the location in the settings menu also
![Screenshot 2024-04-15 123302](https://github.com/community/community/assets/994042/760c8f74-b03a-40be-8341-00a5b3f95b0e)

**Reply by haoliu001** [reply]
Thanks. The problem is that I can not see the "buy button" in my Copilot. And there is only a "Start free trial button", and when I click it, it just refreshes and nothing changes.
![image](https://github.com/community/community/assets/107664290/2624e6a6-0863-4a43-b080-9447f30fac26)

**Reply by KoenBoone** [reply]
strange, are you sure you are logged in with the correct login?
maybe log out and close your browser, then open it again an log back in?
I used up the free trail, then became elligeble for the education license, and all i had there was a button to pay, i clicked it and it automatically activated the education license....

**Reply by KoenBoone** [reply]
also remember, after you applied for the education license and you received the email saying you got it, it took a few days before i could do that.
i also received an email saying that it could take a few days before it would become active and I could add it.
when did you receive the mail that your application for the education license was confirmed?

**Reply by haoliu001** [reply]
It was proved on 8th of this month.

**Reply by KoenBoone** [reply]
hmm, 7 days ago, then it should be active soon, i think mine took also about a week before I could actually activate it.
so keep trying the next few days

**Reply by haoliu001** [reply]
Thanks super much. I will wait another few days and see.

**Reply by KoenBoone** [reply]
keep us informed :-)

**Reply by KoenBoone** [reply]
and? any progress?

**Reply by karhu1** [reply]
how dare you leave a THUMBS DOWN on my post 😥😢😢😢 i updated my original answer hopefully this motivates you to retract your thumb

**Reply by KoenBoone** [reply]
I not only retracted it, but gave it a thumbs up :-)

**Reply by karhu1** [reply]
yippee
